### PR TITLE
1st update

### PR DESCRIPTION
**Pull request by ilikems**
## update caption context

- [ ] check the text
- [ ] check the images
- [ ]  this resolve issue #2

### CODE REVIEW

File: _slides/03-ilikems.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Caption here  New update on Jan 7, 2024
```
Comment (iloveck) [inline _slides/03-ilikems.md:6]: This is looks good, but have you thought about doing this like this:
 .........

Thgis a a change ABC

**Review by iloveck — CHANGES_REQUESTED** [review]

**Comment by ilikems** [comment]
This PR update the copy of ilikems.md file because of out of date

This PR closes / fix / resolve #2

## Done
    - made a commit
    - created a branch
## To Do
 - [ ] merge the branch 
 - [ ]  delete the branch afterwords
 - [ ] cc @ iloveck

File: _slides/03-ilikems.md (hunk 1 above)
Comment (iloveck) [reply _slides/03-ilikems.md]: It looks much better

**Review by iloveck — APPROVED** [review]
Great !

**Pull request merged**